smart-cards/src/main/java/gov/va/api/health/smartcards/patient/PatientController.java

```diff
@@ -95,28 +130,72 @@ void initDirectFieldAccess(DataBinder dataBinder) {
       @Valid @RequestBody Parameters parameters,
       @RequestHeader(name = "Authorization", required = false) String authorization) {
     checkState(!StringUtils.isEmpty(id), "id is required");
-    var credentialTypes = validateCredentialType(parameters);
+    var credentialTypes = getCredentialTypes(parameters);
+    validateCredentialTypes(credentialTypes);
     Patient.Bundle patients = fhirClient.patientBundle(id, authorization);
     Patient patient = getPatientFromBundle(patients, id);
     Immunization.Bundle immunizations = fhirClient.immunizationBundle(patient, authorization);
+    lookupLocations(immunizations, authorization);
     List<MixedEntry> resources = new ArrayList<>();
     consumeBundle(patients, resources, x -> true, this::transform);
     consumeBundle(immunizations, resources, this::filter, this::transform);
```

Comment:
unrelated, but isn't the filtering of completed immunizations already being done in the DQ client now?

Reply:
Yes, this is no longer needed.